configure.ac

```diff
@@ -2034,18 +2061,17 @@ AC_ARG_WITH([cryptoauthlib],
                 trylibatcadir="/usr"
             fi
 
-            LDFLAGS="$LDFLAGS -L$trylibatcadir/lib"
-            CPPFLAGS="$CPPFLAGS -I$trylibatcadir/lib"
-
-            AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <cryptoauthlib.h>]], [[ atcab_init(0); ]])],[ libatca_linked=yes ],[ libatca_linked=no ])
-
-            if test "x$libatca_linked" = "xno" ; then
-                AC_MSG_ERROR([cryptoauthlib isn't found.
-                If it's already installed, specify its path using --with-cryptoauthlib=/dir/])
+            if test "$host_cpu" = "aarch64" ; then
+                LIB_SUFFIX="/aarch64-linux-gnu"
```

Comment:
Is this some sort of cross compile trick for pi? Haven't seen it. Please add comment to .ac.